documentation/docs/concepts/decentralized_identifiers/create.mdx

```diff
@@ -25,17 +25,28 @@ Select your programming language of choice and press the green play button to ex
+
+Updating the identity is possible, which will be discussed in the next section, but all previous states of the DID Document will still be stored in permanodes and any third party recording the network activity.
 
-Next, the identity is created and published to the IOTA Tangle. This operation will generate a private key, storing it in the account, generating a DID, DID Document, and publishing it to the Tangle. Once it is uploaded to the Tangle, it becomes immutable, meaning that this version of the identity can never be altered or removed. The only way to update or delete an identity is by publishing a new version, which we will discuss in the next section. This immutability is what makes a Decentralized Identity solution based on Distributed Ledger Technology (DLT) trustworthy. The public keys inside the DID Document can never be changed without having access to the private key, allowing the users to completely control their own identities. The rest of the example shows how to retrieve (resolve) the identity from the Tangle and how it can be deleted.
 
 ### Identity Generation Process
 
-The generation of an identity starts with a randomly generated asymmetric key pair. This can be generated by the IOTA Identity framework or can be provided as a parameter during the creation process. The public key is hashed using the `Blake2b-256` algorithm. This hash becomes the DID, creating a permanent and provable link between the initial keypair and the DID. The public key is then embedded into the initial DID Document and is used for verifying signatures created with the corresponding private key. This process can be observed and manipulated in depth by using the low-level API available for the IOTA Identity framework. These low-level APIs are available in [Rust](../../libraries/rust/getting_started#api-reference) and [WASM](../../libraries/wasm/api_reference) but are only recommended for complex use cases that require maximum flexibility in the framework.
+In order to create a Alias Output, an existing output is required to make a transaction. For that a new Address is generated and a Basic Output controlled by this address is created using the faucet.
+
+A DID Document is then created which includes one verification method. By now the DID itself is unknown since the Alias Output is not published yet and didn't get an `Alias ID` assigned. For that a placeholder `did:stardust:0x0000000000000000000000000000000000000000000000000000000000000000` is used to reference the DID inside the document.
+
+An Alias Output is created which contains an encoded version of the DID Document in its `State Metadata` and has the state controller and the governor set to the generated Ed25519 address. Note that controllers don't have to be Ed25519 addresses, they can be any type of output. However, they must be unlocked in order perform a state or governance transition when the DID Document is updated or destroyed.
```

Comment:
```suggestion
An Alias Output is created which contains an encoded version of the DID Document in its `State Metadata` and has the state controller and the governor set to the generated Ed25519 address. Note that controllers doesn't have to be a Ed25519 addresses, they can be any type of output. However, they must be unlocked in order perform a state or governance transition when the DID Document is updated or destroyed.
```

Reply:
Grammar is not correct here, since `controllers` and `addresses` are plural.

Reply:
you are right, disregard the suggestion.